crates/experimentation_platform/src/api/experiments/handlers.rs

```diff
@@ -872,9 +897,21 @@ async fn update_overrides(
         ))
         .get_result::<Experiment>(&mut conn)?;
 
-    let mut http_resp = HttpResponse::Ok();
-    add_config_version_to_header(&config_version_id, &mut http_resp);
-    Ok(http_resp.json(ExperimentResponse::from(updated_experiment)))
+    match execute_webhook_call(
+        tenant_config.experiments_webhook_config,
+        updated_experiment.clone(),
+        &config_version_id,
+        WebhookEvent::ExperimentInprogress,
```

Comment:
I you can't update_overrides while the experiment is inptogress, so you can propogate created here